src/devices/Interop/Unix/Libc/Interop.open.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [DllImport(LibcLibrary, SetLastError = true)]
+    internal static extern int open([MarshalAs(UnmanagedType.LPStr)] string pathname, FileOpenFlags flags);
```

Comment:
Why can't you use FileStream and just get a [handle](https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.handle?view=netframework-4.8) out of it?

Reply:
I referred to the implementation of `I2C`, and Interop was copied from System.Device.Gpio project.

Reply:
ok, we should clean duplicate code in the future but fine in this PR